src/controllers/auth/index.ts

```diff
@@ -0,0 +1,4 @@
+import { login } from './login';
+import { logout } from './logout';
+
+export { login, logout };
```

Comment:
i don't think this pattern of exporting individual methods is necessary - why not just have a single auth file? that way it matches the pattern that we have for other controller files